oasislmf/pytools/aal/cli.py

```diff
@@ -6,6 +6,11 @@
 from . import manager, logger
 
 
+def validate_flags(args):
+    if args.binary and args.parquet:
+        raise RuntimeError("Cannot output both parquet and binary flags at the same time.")
```

Comment:
If we only allow one output, can we have just one arg for example [-f, --output-format] that take csv, bin or parquet as value to decide on the format of the output?
It would differ from the c++ version but we already have a specific command I think its ok